config/sidekiq.yml

```diff
@@ -1,8 +1,5 @@
 ---
 :queues:
   - default
-  - mailers
-  - active_storage_analysis
-  - active_storage_purge
```

Comment:
As of Rails 6.1, these separate queue names are not needed. All jobs now use "default".